queries/headers/suspicious_recipient_pattern.yml

```diff
@@ -0,0 +1,11 @@
+name: "Suspicious Recipient pattern"
```

Comment:
Let's consider updating this to be more descriptive until Descriptions are shown in the UI, e.g. "'To' matches 'From' but not 'Return-Path'"